dev-python/kiwisolver/files/kiwisolver-1.4.4-pypy3-tests.patch

```diff
@@ -0,0 +1,29 @@
+url: https://github.com/nucleic/kiwi/issues/157
```

Comment:
Link to commit always too, please :)

Reply:
(or PR...)

Reply:
Sorry, I didn't submit a merge request, I provided a trivial patch in the issue. I don't have more (for now). This is not good enough?

Reply:
Ideally just throw up a quick PR upstream so we don't have to carry it forever, but it is indeed trivial.

Reply:
Done.